Package.swift

```diff
@@ -1,12 +1,12 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
```

Comment:
We could leave this file unmodified and create a separate one named `Package@swift-5.7.swift`, but it's probably okay to drop Xcode 13 support already.

Reply:
Yeah, that was initial idea, but then it's a duplicate and `swift build` was failing. So I decided it worth to update xcode requirement, as it xcode 13 is not supported in appstore anyway.